public/pull_request_template

```diff
@@ -0,0 +1,20 @@
+## Describe your changes
```

Comment:
Maybe we could split this section into two:

- Summary
- Details

In `Summary`, we could describe the changes made in 1-2 sentences as a TLDR. In `Details` we can be more specific and break down our changes in sub-sections based on our specific changes.

This way, we both have a quick overview and a detailed section. What do you think?

Reply:
We could potentially also add sub-sections (which are optional) to the `Details` section like:

- Challenges (what problems did I face?)
- Alternative implementations (what alternatives were considered and why were they disregarded?) 
- Solution (how did I fix the problems?)
- Implementation details (how was this change implemented?)

We could add a comment to all these sections that they are optional!